import-service/importProductsFile.js

```diff
@@ -0,0 +1,38 @@
+const AWS = require('aws-sdk');
+const { headers } = require('./headers');
+const { IMPORT_BUCKET_NAME } = require('./constants');
+
+module.exports = async (event) => {
+    const s3 = new AWS.S3({region: 'eu-west-1'});
+    const {name} = event.queryStringParameters;
+    const destinationFileName = `uploaded/${name}`;
```

Comment:
Since you operate with the folder name It would be nice to create an appropriate constant.
https://github.com/handleman/nodejs-aws-be/pull/3/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.yml&file-filters%5B%5D=dotfile#diff-49e65129f7beb58ca2572f8db7b5d6ae67fcd556ff53156f461064b71af0a366R19